### ISSUE TITLE
binance orders can't be canceled 

### ISSUE THREAD

**Issue by nshopik** [ctubio/Krypto-trading-bot#1159]
maybe binance did change something, but K can only push new orders but can't delete them

**Comment by secwang** [comment]
yes ,same error, some time K stuck on deleting order,have to restart every 1H。

**Comment by secwang** [comment]
I understand the situation, and the conclusion is that I need to write a script to clean up zombie orders so that I can keep running.
The logic is simple: the order continues to exceed the 1min (too long for high frequency) and we cancel the order.

**Comment by secwang** [comment]
I am using such a strange way to solve this problem at present. Use a python script to check if Qian'an has a zombie order, and if so, restart k. So far, the job is very good.

**Comment by hjbc3** [comment]
got same issue here , when can this bug be fixed

**Comment by Vangervp** [comment]
Same issue. Please fix bug with Binance,
